common/src/test/java/gov/cms/ab2d/common/service/CacheServiceImplTest.java

```diff
@@ -117,7 +118,7 @@ void given_contractNumber_and_month_should_clear_cache() {
         cut.clearCache(request);
 
         CoveragePeriod coveragePeriod = coveragePeriodRepo.getByContractIdAndMonthAndYear(contract.getId(), january, YEAR);
-        final List<String> activePatientIds = coverageRepo.findActiveBeneficiaryIds(coveragePeriod);
+        final List<String> activePatientIds = coverageRepo.findActiveBeneficiaryIds(Collections.singletonList(coveragePeriod));
```

Comment:
Static imports are really nice here, especially since singletonList is the only method used out of Collections.